sherpa/optmethods/__init__.py

```diff
@@ -372,8 +372,8 @@ class LevMar(OptMethod):
        free parameters.
     epsfcn : number
        This is used in determining a suitable step length for the
-       forward-difference approximation; default is sqrt( DBL_EPSILON
-       ) ~ 1.19209289551e-07, where DBL_EPSILON is the smallest number
+       forward-difference approximation; default is FLT_EPSILON ~ 1.19209289551e-07,
+       where DBL_EPSILON is the smallest number
```

Comment:
I should note that by "80-column limit" I really mean "a somewhat random limit that can vary per docstring and depends on what version of emacs and pyflakes/pep8 I have available, but is somewhere near 70-80 ish".